fmc/src/hand_off.rs

```diff
@@ -97,6 +124,10 @@ impl HandOff {
         }
     }
 
+    /// Retrieve image manifest load address in DCCM
+    pub fn image_manifest_address(&self, _env: &FmcEnv) -> u32 {
+        self.fht.manifest_load_addr
```

Comment:
Check if ptr is within bounds of the dccm

Reply:
done